cmd/search_test.go

```diff
@@ -164,7 +164,7 @@ func TestSearchRun(t *testing.T) {
 	}))
 	rc := regclient.NewClient(&regclient.Config{Location: server.URL})
 
-	f, err := NewTestFactoryInstanceOptions(ctx, lib.OptRegistryClient(rc))
+	f, err := NewTestFactoryInstanceOptions(ctx, run.RootPath, lib.OptRegistryClient(rc))
```

Comment:
This looks a bit confusing, but works okay. The intention with TestRunner is that the RootPath is *not* the RepoPath, but rather the parent directory that everything temporary lives inside of (and this difference has bitten before, causing subtle bugs in tests). However, TestRunner in general needs some cleanup, documentation, and clarification of intention. Could you add a TODO here to add an explicit RepoPath to the SearchTestRunner?